css/styles.css

```diff
@@ -55,20 +60,239 @@ header .menu-button {
   color: #36b37e;
 }
 
-.headline p {
-  margin-top: 3em;
+.paragraph-text {
+  font-size: 1.2em;
   line-height: 1.5em;
   text-align: center;
   color: #172b4d;
 }
 
+.headline .paragraph-text {
+  margin-top: 3em;
+}
+
 .headline ul {
   width: 100%;
   margin-top: 3em;
   display: flex;
+  -ms-flex-pack: center;
   justify-content: center;
 }
 
+.work-card img {
+  width: 100%;
+}
+
+.knowledge-icon-container img {
+  width: 2.8em;
+}
+
 .headline ul img {
   margin: 0.5em;
 }
+
+.recent-work-section {
+  width: 100%;
+  display: flex;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+
+.section-headline {
+  padding: 0 1em;
+  font-size: 2.5em;
+  text-align: center;
+  font-weight: bold;
+  color: #172b4d;
+}
+
+.recent-work-section .section-headline::after {
+  display: block;
+  margin: 0.5em auto;
+  content: "";
+  width: 1em;
+  height: 0.1em;
+  background-color: #36b37e;
+  border-radius: 0.25em;
+}
+
+.work-cards-container {
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-columns: repeat(auto-fit, minmax(320px, 350px));
+  grid-template-columns: repeat(auto-fit, minmax(320px, 350px));
+  -ms-flex-pack: center;
+  justify-content: center;
+  grid-gap: 1.5em;
+  width: 100%;
+  padding: 0 1.5em;
+}
+
+.work-card {
+  width: 100%;
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-rows: 55% 45%;
+  grid-template-rows: 55% 45%;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 0.5em;
+  overflow: hidden;
+}
+
+.work-card-info {
+  display: flex;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  padding: 0.5em;
+  background-color: white;
+  border-top-left-radius: 0.5em;
+  border-top-right-radius: 0.5em;
+}
+
+.work-card-info h3 {
+  padding: 0 1em;
+  font-weight: 600;
+  color: #3a4a42;
+  text-align: center;
+  font-size: 1.5em;
+}
+
+.work-card-info ul {
+  margin: 1em 0;
+  display: flex;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.work-card-info ul li {
+  font-size: 0.8em;
+  font-weight: 600;
+  padding: 0.8em 1em;
+  background-color: #ebf0ee;
+  color: #3a4a42;
+  border-radius: 0.3em;
+}
+
+.green-button {
+  padding: 0.8em;
+  font-size: 1em;
+  color: white;
+  text-decoration: none;
+  border: none;
+  background-color: #36b37f;
+  border-radius: 0.2em;
+}
+
+.work-card-info .green-button {
+  margin: 0.8em auto 2em auto;
+  max-width: 8em;
+}
+
```

Comment:
- [ ] Please the button should  follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468))
![image](https://user-images.githubusercontent.com/74744787/159027829-7c64b102-0a9f-4fdf-afaf-7992c72692b3.png)